setup.py

```diff
@@ -35,6 +35,8 @@
 
 # NApps enabled by default
 # CORE_NAPPS = ['of_core']
+os.environ["KYTOS_TESTS_SIZE"] = 'small'
+os.environ["KYTOS_TESTS_TYPE"] = 'unit'
```

Comment:
Shouldn't it be `os.environ.get("KYTOS_TESTS_SIZE", 'small')`?

Reply:
Solved moving the default value to decorator.